ros2/source/source/ros_entrypoint.sh

```diff
@@ -2,5 +2,5 @@
 set -e
```

Comment:
This `ros_entrypoint.sh` file is no longer used by the `source` Dockerfile, as well as made redundant given the parent image, `devel`, already includes an approrape entrypoint.

Reply:
:+1: good catch, this is a leftover, it is not used by the new template. I deleted the source and devel folders to make sure to remove any deprecated files in https://github.com/osrf/docker_images/pull/246/commits/897cf8408b46c0edf5ce0781c4b4644cb6f7fd82